### DISCUSSION TITLE
Tree preservation orders

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#43] [Planning consideration]
# Tree preservation orders

A [Tree Preservation Order](https://www.gov.uk/guidance/tree-preservation-orders-and-trees-in-conservation-areas) is an order made by a local planning authority in England to protect specific trees, groups of trees or woodlands in the interests of amenity

## What

We have drafted [a specification for tree preservation orders](https://digital-land.github.io/specification/specification/tree-preservation-order/). It allows data providers to create digital records of their tree preservation orders, whether they cover a zone (such as woodland) or individual trees.

Currently, we are testing it with a number of Local planning authorities to see if it works for them. We are testing whether they can publish data that complies with the standard and whether the data produced contains the right valuable information.

## Why

Tree preservation orders place restrictions on what can be done to the tree, for example, cutting it down might be prohibited. This has an impact on development. If there is a tree, on a site, with a tree preservation order the development that will be deemed acceptable is different to a site without a tree preservation order. Therefore, it is important for anyone wishing to develop a site to know if there is a tree preservation order and what it covers.

If there was a complete digital record of tree preservation orders, software could be built to more easily determine if a site or development is impacted.

## What we're looking for

We'd like to know more about:

* if the standard includes all the data you'd expect it to include? If not, tell us what you think is missing?
* what would you use the data for?
* is there existing tree preservation order data?
* can you publish data that matches the standard?
* anything else we need to know about tree preservation orders

**Comment by mattlucht** [comment]
We've had a question from one the Open Digital Planner LPA partners about dates relating to TPOs:

`TPOs have in effect two start dates. The date they are “made” and therefore come into force for a period of 6 months and the date they are “confirmed” which is the date they become permanent.  The dates we have at present are confirmed dates for confirmed TPOS and made dates for unconfirmed TPOS. If we are to make reference to ‘made date’ there will be a bit of work to go through historic files to find out the date the orders were originally made. It would useful to have some clarification here.`

It's be interesting to know if this also applies to other LPAs?

**Comment by colmjude** [comment]
Also, it would be useful to know more about what these dates mean, if they used to make certain decisions and if so, what those decisions are.

For example, does a TPO have to be 'confirmed' before it affects a potential planning application?

**Comment by mattlucht** [comment]
I had a conversation with some of the team from Great Yarmouth – they've advised that a TPO that was `made` would have the same implications from a planning perspective as a TPO that is `confirmed`.

At present in their data the `made` date is overwritten with a `confirmed` date when the TPO hits that status. They're going to add two date fields `made-date` and `confirmed-date`.

For older TPOs they won't have the data, but this shouldn't be a problem. It will however be useful for new TPOs where in the case of an appeal it'd be important to not lose the `made` period.

**Comment by mattlucht** [comment]
Based on the above, I propose we look at the dates in the specification and potentially include a `made-date` and a `confirmed-date`. In the case of there only being one date present, would it make sense that we use that as the `confirmed-date`?

**Reply by RichLimbrick** [reply]
@mattlucht My understanding of TPOs is a bit sketchy, but I think it's important to be very clear which one it is where there is only one date on record. The MADE date means there is provisional protection from that date for a period of six months. The direction has to be CONFIRMED for it to have permanent protection. That provisional protection given by the section 201 direction comes to an end after six months, although the LPA are not prevented from confirming the TPO after the six month period.

That means:
if there is only a made date, and it's less than six months old, the tree is protected
if there is only a made date, and it's more than six months old, the tree is not protected
if there is a confirmed dare, the tree is protected

**Reply by mattlucht** [reply]
thanks @RichLimbrick this is helpful! :)

**Comment by mattlucht** [comment]
Based on discussion with various LPAs we're proposing to make some changes to the [tree preservation order specification](https://digital-land.github.io/specification/specification/tree-preservation-order/).

We're proposing to add:

* `made-date`
* `confirmed-date`
* and to the `tree-preservation-zone` we'll update `tree-preservation-type` to `tree-preservation-zone-type` that includes `area`, `group` and `woodland`
* `point`, but its worth noting that if a `geometry` is available then we'll determine the `point` from that

**Reply by colmjude** [reply]
We have made these proposed changes and bumped the version to [1.2.2](https://digital-land.github.io/specification/specification/tree-preservation-order/)

**Comment by mattlucht** [comment]
Also based on conversations with LPAs we have a few questions:

* we've heard that `tree-species` might be important. It would be helpful to know how the species of tree is used in relation to TPOs. It'd also be useful to know whether there is an agreed list of tree species that we can point to?
* UPRNs, do tree's have a UPRN?
* when a tree preservation order zone is defined, do the trees protected within that order change?

**Reply by psd** [reply]
On the UPRN, quite a few TPOs we looked at have addresses for the trees, so I think we included the address-text and UPRN to model those values where they exist, but not something needed for every tree.

**Reply by psd** [reply]
On tree species, we have seen some tree datasets, such as the [Local Authority Maintained Trees](https://data.london.gov.uk/dataset/local-authority-maintained-trees) list on the London Datastore which have a text name for the tree species. Ideally we'd use an open taxonomy such as [Forest Research](https://www.forestresearch.gov.uk/tools-and-resources/tree-species-database/) so providers can categorise a tree with the general family, genus or specific species, where it's known and available.

**Comment by psd** [comment]
We have a number of LPAs wanting to provide a geometry (shape) for individual trees, or who don't track individual trees. Should we combine the [tree](https://www.planning.data.gov.uk/dataset/tree) dataset into the [tree-preservation-zone](https://www.planning.data.gov.uk/dataset/tree-preservation-zone) dataset, and add a [tree-preservation-zone-type](https://www.planning.data.gov.uk/dataset/tree-preservation-zone-type) of `individual-tree`?

**Reply by mattlucht** [reply]
We've been thinking about how we might handle combining the two datasets. Assuming that this is the right thing to do here's an initial thought on what the data might look like.

It feels like the dataset will need a new name as neither `tree-preservation-order-zone` or `tree` seem to make sense. Perhaps `tree-preservation-order-location`?

|field-name|comments|
|-----------|----------|
|reference||
|name||
|tree-preservation-order||
|uprn|URPNs wouldn't apply to areas only to specific trees|
|address-text|Similar to URPNs, would address-text only apply to an individual tree?|
|point||
|geometry||
|tree-preservation-location-type|this would include `group`, `woodland`, `individual-tree`|
|notes||
|organisation||
|entry-date||
|start-date||
|end-date||

**Reply by colmjude** [reply]
Thinking about how data providers would produce the data it might not be right to include a field like `uprn` that is only used in some circumstances (i.e. when it is an individual tree). Could zones/areas have a UPRN?

Thinking about data consumers using the data, emerging the datasets would add an extra step for users wanting to only use zone data or only use data for individual trees. To understand if this trade-off is worth us emerging the datasets we need to understand how people want to use the data. 

Please share any examples of use cases or scenarios where you could need the data for all things covered by TPOs or just the zones or just trees.

**Comment by colmjude** [comment]
### Revoking and felling trees

In some recent conversations with planning authorities, we learnt that
* TPOs can be revoked - once revoked the TPO is no longer available publically
* individual trees can be felled via "consent work" - in these cases planning authorities do not always have the resources to revoke and issue the affect TPOs

We'd like to understand more about the processes involved.

Revoking TPOs
* what is the process for revoking a TPO?
* under what circumstances does a planning authority revoke a TPO?
* how often for planning authorities revoke TPOs?
* do they keep copies of revoked TPOs?

Felled trees
* what is "consent work"?
* what is the process someone must go through to fell a tree?
* are there artefacts produced during this process?
* does this process only work on individual trees or before groups of trees?

If anyone has any information they could share with us on these topics that would be great.

**Comment by kieranwint** [comment]
## OS Update

We have been doing some work with Ordnance Survey on giving LPAs more confidence in what they are publishing, through making explicit what is covered under the terms of the PSGA.

As a result, they have recently updated their [website](https://www.ordnancesurvey.co.uk/customers/public-sector/public-sector-licensing/publish-derived-data), under the “planning” sub-heading, to state that “tree preservation orders as a single point or tree preservation zone/group depicted as a polygon” fall under the terms of the presumption to publish criteria.

This falls in line with how we expected the data to be produced (point data for tree, polygon data for tree-preservation-zone), and means that if data is made to our technical specifications then it is shareable under the open government licence.

If you have any further thoughts or concerns, please reach out!

**Comment by MrMikeRose** [comment]
As part of our mission to build the foundations that will enable the planning system to transform, we want to find out from LPAs what is possible in terms of data collection and provision to satisfy the current technical specifications.

For the last few months we have been running a working group with a selection of local planning authorities (LPAs) to continue to refine our [data specification](https://digital-land.github.io/specification/specification/tree-preservation-order/)  and develop our guidance on tree preservation orders (TPOs).

The aim of the group is to help MHCLG understand the challenges faced by LPAs and therefore allow us to design the guidance that enables LPAs to deliver the TPO data in accordance with the specification. 

We have held discussions around some of the key challenges faced by LPAs, such as trees being mapped incorrectly, trees having been removed without consent, new trees not being covered by a TPO, some LPAs consider ‘exemptions’ pending revocation where dead trees have been felled, and also what requirements HMLR have for the data for the Local Land Charges Register.

We then held a focused session on the processes involved in making and revoking a TPO, to help us understand the amount of effort required to complete these processes. The highlight from this is the manpower required to complete site visits, urgent requirements for a TPO, the resources required to review existing TPOs as a desktop exercise and a potential for a high volume of protected trees having been felled without permission.

Finally, we held a session focused on the revocation of a TPO, as this aspect appears to be one of the headaches for LPAs. We explored why TPO data is ‘deleted’ by LPAs rather than given an ‘end date’ as our current data specification requires. We then mapped the revocation process, which helped us to understand the process can be complex, requiring a lot of coordination between departments and officers. We also explored TPOs pending revocation, and what happens which this is pending.

MHCLG now have a few actions to take forward and get answers to before we reconvene the working group. These are:

- What data do HMLR currently require, and how does this align with our data requirements?
- What are the legal requirements when revoking a TPO? What data, if any, still needs to be available?
- What can we do to make the data easier to provide?

We will reconvene the group in a few weeks time for a session to discuss the answers, which may involve further exploration into the findings.

**Reply by MrMikeRose** [reply]
Thanks to Joe Hasker-Sarchet for drafting this note ^

**Comment by kieranwint** [comment]
Hi all - just to provide a quick update on where the data design team are with regards to the TPO specification:

- Based on the feedback we gathered from the previous working group sessions, we are currently working on factoring this into an updated and revised specification.
- We will be holding another working group session to present back the revised specification on Thursday 17th October at 11:00 - if you would like to attend please let us know.
- In the meantime, please do continue to work through producing and providing your TPO data. We have had a look at the LLC schema but the early indications are that our TPO specification and the ask of the LLC register are sufficiently different to warrant keeping them separated.

If you have any further questions please do not hesitate to get in touch with us.

**Comment by colmjude** [comment]
## Data specifications updates

A few months ago we had some questions about [tree preservation orders](https://github.com/digital-land/data-standards-backlog/discussions/43#discussioncomment-9988259), these were things that had been coming up in conversations with local planning authorities.

In the last few months, we've been running the working groups that [Mike and Joe have documented above](https://github.com/digital-land/data-standards-backlog/discussions/43#discussioncomment-10533975).

These have helped us know where we need to tweak the specification and provide further clarification.

**Reply by colmjude** [reply]
### Revoking TPOs

Some questions and queries came up in the working groups about how to handle revoked TPOs. We’ve been giving this some thought and checking the legislation

For now, please use the `end-date` to express when the TPO is revoked - it should be the date it was revoked.

At this stage in the lifecycle of a TPO, our understanding of the legislation is that, LPAs are no longer obligated to keep the original TPO document publically available, therefore, if you are removing it from public view then we’d expect the `document-url` field to be blanked out, removing the url to the original document from the record. 

Doing this means users of the data will know a TPO used to exist but has now been revoked but they will not be able to access the original TPO document.

**Why:** This is a safer approach than removing the record completely because records can go missing by accident. This approach will avoid any confusion. If we just remove the record, then users may wonder, “Has this record that used to be here been revoked, or has it disappeared accidentally?” and attempt to contact us or the responsible authority. 

Therefore we should keep the record but with an `end-date` and without a link to the legal document.

---
One thing we’d like further clarification about.

We read in the legislation that LPAs are supposed to
```
11.  Where an authority revoke an order they shall—
    (a) endorse the original order with a statement to the effect that the order has been revoked, specifying the date of the revocation;
```
Do LPAs issue statements informing the public/landowners that a TPO has been revoked? Are these statements published on an LPA site?

**Reply by colmjude** [reply]
### Felled trees

Lots of planning authorities raised points about felled trees.

We know:
* felled trees are important. For both planning authorities administering TPOs and citizens/developers needing to know they are compliant with the TPO.
* felled tress can be quite prevalent. We were told that in some cases up to 30-50% of trees covered by a TPO have been felled.

We don't know:
* much about the process around felling trees. How that happens, how it is carried out and how it is recorded

This suggests at minimum we need to record if and when a tree was felled. To do this we are going to add a `felled-date` field to the `tree` dataset.

How to use this field:

* if a tree has NOT been felled, the field should be left blank.
* if the exact date the tree was felled is available then put that. The more precise the better
* if a rough date is known, for example, the month and/or year it was felled then put that.

In both cases, it has been felled; the presence of a value in the `felled-date` field indicates the tree has been felled. Therefore anyone using the data can confidently know which trees were part of the TPO but have been felled.

---
In terms of testing whether the change to the specification is all that is needed, we need to understand more about felling trees.

We need to find out:
* what is the process for felling a tree?
* is this done by work commissioned by the local planning authority or can it be done by 3rd parties? 
** if it is done by the local authority how is the work triggered and then carried out? Are there defined processes and any records?
** if it is done by 3rd parties, how is it done by other parties, do they have to ask permission or apply? How is that done?
* what happens in the case of a tree being felled because it has died?
* what happens if an LPA finds out a tree has been felled illegally?
* a few LPAs also mention that some felled trees are “replaced” when new trees are planted. It would be good to understand more about this too.

**Reply by colmjude** [reply]
### Additional information relating to TPOs

 Some LPAs have informed us they would like to provide some additional information about TPOs or specific trees. For example, saying they are aware there is no longer a tree, and have no intention of prosecuting in relation to the removal of the tree. 

This can be achieved by using the `notes` field.

We recommend using this field. If and when planning authorities start using this field we can do some analysis to see if it is being used in similar ways - that would then require further work to understand if the data specification should change.

**Comment by colmjude** [comment]
## Why is one of the datasets called tree?

Recently we were asked why the `tree` dataset is called `tree`?

It is a good question and was an intentional decision. Here is the explanation we gave.

We take a relational database-style approach to data modelling which means we try to break each “thing” (for example, conservation area or tree preservation order) we look at into its constituent parts.

In the case of tree preservation orders this means we’ve split it into 3 datasets (or parts). We have

* `tree-preservation-order` dataset - this is used to record the official tree preservation order document or legal instrument
* `tree-preservation-zone` - this is used to record the specific areas or zones protected by the tree preservation order
* `tree` - this is used to record the individual trees that may be protected by the tree preservation order

Hopefully, that helps to explain the idea behind having 3 datasets for tree preservation orders.

In terms of naming or labelling it `tree`, that’s a good question and something we’ve not explained very well.
We chose `tree` because the dataset could represent a collection of all individual trees.

Unlike `tree-preservation-zones`, which can be zones designated explicitly for the purpose of the tree preservation order, we felt that wasn’t the same for trees. A tree exists outside of a tree preservation order. It exists first outside of a TPO and then can be covered/protected by a TPO (or not). We capture this by referencing the TPO, we use the `tree-preservation-order` field of the tree dataset to do that. If there is a value in this field it is protected.

So, although we are only asking LPAs (or anyone) to provide tree data for trees referenced in tree preservation order, the tree dataset could handle any tree (see attached image).

* Protected trees (covered by a TPO) would have a value (a TPO reference) in the tree-preservation-order field.
* Unprotected trees would have a BLANK tree-preservation-order field.

![trees-dataset](https://github.com/user-attachments/assets/725564cd-bdb1-4f45-a17b-0a86d0ec6863)

**Comment by kieranwint** [comment]
### Tree preservation orders - guidance update

In the middle of October, we held our final working group session for tree preservation orders. In this session, we discussed the proposed changes we were to make to the specifications, the summary of which can be seen above in the post from October 14th 2024.

As part of our next steps from that session, we mentioned we would ensure that our amendments to the specification were reasonable in line with the legislation surrounding tree preservation orders, and as a result we are confident that what we are asking for in the specification aligns with the [existing legislation](https://www.legislation.gov.uk/uksi/2012/605/contents). 

We also cross-referenced our TPO specification with the HMLR LLC schema to see how closely aligned they are. For decision-making in planning, we require more data points for TPOs, therefore, it makes sense to keep the HMLR and MHCLG specifications separate.
We have also recently [updated the guidance](https://www.planning.data.gov.uk/guidance/specifications/tree-preservation-order) for the tree preservation order datasets to match the changes we made to the [specification](https://digital-land.github.io/specification/specification/tree-preservation-order/).

If there are any concerns regarding our ask in the specification, or anything unclear within the guidance, please do not hesitate to drop us a line here!

**Comment by Swati-Dash** [comment]
@colmjude Please see this message from Southwark, we currently don't have a a tree-preservation-zone-type as "individual". Many LPAs have provided us this category and we have added them in the Tree dataset as they were individual trees. Please see the explanation from Southwark on the definition of "Individual" tree-preservation-zone-type. 

The Individual zone designation aligns with the land title for each site, this is to identify land charges applicable for particular properties. The TPO document itself includes the map and schedule for the individual trees, in line with the regulations. Our GIS system is designed to alert customers to the fact that there are protected trees on site, it is then up to the applicant to review the ‘free to view’ document and identify the tree(s). 

Whilst it may be preferable to have a distinct TPO made on each property this has not always been the case and some TPOs cover several addresses. The Council are not required to remake Order once confirmed and the ability to do so relies heavily on departmental budgets as tree works applications are not subject to planning fees. 

The polygon may cover several properties, particularly where there has been back land or infill development since the order was first made. 

Our Land Charges Team will add or remove addresses as orders are varied or revoked. Where an Order is revoked, the polygon would be removed along with the Land Charge and (where applicable) a new order made for replacement planting and a new zone added. 

Kind regards, 

Senior Planner – Urban Forester 

Design, Conservation and Transport 
Planning & Growth | Resource Department

**Comment by greg-slater** [comment]
Our specifications for [tree](https://github.com/digital-land/specification/blob/12d4bbe8f8855b8377eca3951cf85d777462f45e/content/dataset/tree.md?plain=1) and [tree-preservation-zone](https://github.com/digital-land/specification/blob/12d4bbe8f8855b8377eca3951cf85d777462f45e/content/dataset/tree-preservation-zone.md?plain=1) respectively contain the placeholder fields `tree-species` and `tree-species-list`, which are not mentioned in the [technical specification](https://digital-land.github.io/specification/specification/tree-preservation-order/) or [guidance](https://www.planning.data.gov.uk/guidance/specifications/tree-preservation-order); this is because the requirements for this species data aren't fully understood at the minute.

However, we've agreed that where LPAs do publish species, we'll map it to these placeholder fields as it will allow us to track the prevalence of species data, and means that data consumers will be able to access the data that is available.

Peterborough LPA provided the following comment on the need for species data:

> It is essential to know the species in all respects to identify if a tree is protected. For example, many residents have historic TPOs on their property, but the protected tree has been felled and the council's records not updated. Future residents may plant trees in their garden close to the original TPO location, but these will not be protected. If a resident or planner uses your GIS dataset, they may assume those trees aligning with the original TPO plot are protected, which is not the case.

**Reply by paris-dp** [reply]
During today's drop in, Peterborough LPA also raised that the felled field 'felled-date' being in date format can be challenging as it is not always known how long a tree has been felled for.

**Comment by drewhardy77** [comment]
Conversations with Croydon LPA today highlighted a couple of user needs, and some potential mitigations. These are not necessarily limited to TPO data and could have wider applicability. 

1. LPAs need to have a way of indicating whether a TPO record has been verified or not, and potentially how it has been verified e.g. direct observation, planning record, remote sensing....
Suggested methods of identifying within the data:
- add field `last-reviewed-date` (_nb could we use `entry-date` for this purpose? I am keen not to add any more dates into the spec_)
- add field for `record-status` to identify whether a record is verified or unverified. Standard enum list rather than free text. Is there a need for anything more than `verified` and `unverified`?
- add field `verification-method` to understand how a verified record was verified e.g. remote sensing, planning application , site visit etc. This may 

**Impact of doing nothing**

Publishing data 'as-is' may give a false impression of certainty, and lead to misuse of the data, whether intentionally or unintentionally. For example, if a user sees no record of a protected tree on a site, but a TPO exists and has been inaccurately recorded, can the user point to the data and say they were justified in removing the tree as it did not appear to be protected?

Working on the premise that LPAs may have a significant volume of TPO data to verify, if we lack a means for identifying whether a record has been verified or not (or when) this leads to LPAs being reluctant to publish until everything is verified, which in practice means they are unlikely to publish at all.
